README.md

```diff
@@ -11,3 +11,30 @@ is potentially incompatible with Trusted Types, tsec emits compilation errors.
 
 tsec is based on the open source TypeScript static analyzer
 [Tsetse](https://tsetse.info/).
+
+## Build and run
+
+### Build
+```
+  # Clone the repository
+  git clone https://github.com/googleinterns/tsec
+  cd tsec
+  
+  # Install gulp
+  npm install gulp
```

Comment:
Why do we need gulp? It works for me even if I skip this step. If we need it it should probably be in `devDependencies`

Reply:
We don't need gulp. I guess Anna added this because she works on VSCode which requires gulp